quipucords/scanner/network/utils.py

```diff
@@ -14,10 +14,23 @@
 from ansible.parsing.utils.addresses import parse_address
 from ansible.plugins.inventory import detect_range, expand_hostname_range
 
+from api.models import ScanJob
 from api.vault import decrypt_data_as_unicode
 
+from scanner.network.exceptions import (NetworkCancelException,
+                                        NetworkPauseException)
+
+
+def check_manager_iterrupt(interrupt_value):
```

Comment:
change the name of this function to `check_manager_interrrupt`

Reply:
interrrupt 🤔

Reply:
🤔